mingw-w64-git/git-wrapper.c

```diff
@@ -113,15 +117,56 @@ static void my_path_append(LPWSTR list, LPCWSTR path, size_t alloc)
 	}
 }
 
+bool IsRunningOnARM64()
+{
+	// Note: ARM64 support as well as IsWow64Process2 are only available in Windows 10 1511+
+	if (!IsWindows10OrGreater()) {
```

Comment:
Do we even need this check here? It seems redundant since you check the API availability anyway.

Reply:
Correct, I added it just to be 100% sure we don't create any conflicts on older Windows versions. Happy to remove it though

Reply:
AFAIK this helper function uses GetVersionEx under the hood, which is discouraged to my knowledge (because of "version lies" etc.) and the API check is suggested as a replacement:

https://docs.microsoft.com/en-us/windows/win32/sysinfo/operating-system-version

I might be totally wrong in light of the Git for Windows policies though, but it looks like this check doesn't add much value here 🙂

Reply:
You're right, thanks for linking the docs:

> You can test for the presence of the functions associated with a feature. To test for the presence of a function in a system DLL, call the LoadLibrary function to load the DLL. Then call the GetProcAddress function to determine whether the function of interest is present in the DLL

So the combination of `GetProcAddress()` and `GetModuleHandle()` should be absolutely fine here. Before I added the `IsWindows10OrGreater()` check, I could also confirm that everything was working correctly in Windows 8.1 as well. I'll remove the additional check 👍🏼